README.md

```diff
@@ -52,6 +52,7 @@ Modern software development practices _assume_ support for reviewing changes, tr
 - [Gitkube](https://gitkube.sh/) - Build and deploy docker images to Kubernetes using git push
 - [Jenkins X](https://jenkins-x.io/) - a CI/CD platform for Kubernetes that provides pipeline automation, built-in GitOps and preview environments
 - [Weave Cloud](https://www.weave.works/product/cloud/) - GitOps experience, workflows and dashboard for your cluster(s) (commercial product from Weaveworks)
+- [Werf](https://werf.io) - GitOps tool with advanced features to build images and deploy them to Kubernetes. Integrates with any existing CI system.
```

Comment:
the style guidelines of awesome-list dictates that all of the items should have a trailing period or none of them should.  thusfar we've gone with "none of them".  This entry breaks that.  I don't frankly care too too much one way or the other, but I would prefer if you could just follow what we already have in place.
```suggestion
- [Werf](https://werf.io) - GitOps tool with advanced features to build images and deploy them to Kubernetes (integrates with any existing CI system)
```